config/settings/base.py

```diff
@@ -609,6 +609,20 @@
     },
 }
 
+# Time estimates for reviewing tasks based on the assigned story points
+#
+# SPRINT_HOURS_RESERVED_FOR_REVIEW {
+#     "null": Review time if task is not estimated,
+#     "1.9": Review time if task has less than 2 story points,
+#     "2": Review time if task has 2 story points,
+#     "3": Review time if task has 3 story points,
+#     "5": Review time if task has 5 story points,
+#     "5.1": Review time if task has more than 5 story points
+# }
+# Any time estimate that is not defined here, will use the "review time" from the closest value defined
+
+SPRINT_HOURS_RESERVED_FOR_REVIEW = json.loads(env.str("SPRINT_HOURS_RESERVED_FOR_REVIEW",
+                                                      '{"null": 2, "1.9": 0.5, "2": 1, "3": 2, "5": 3, "5.1": 5}'))
```

Comment:
Wouldn't it make sense to use actual values instead of strings as keys? I know that it's a limitation of JS, but there are two workarounds:
1. Create something like this:
    ```python
    def jsonKeys2int(x):
        if isinstance(x, dict):
            res = {}
            for k, v in x.items():
                try:
                    new_key = float(k)
                except ValueError:
                    new_key = None
                res[new_key] = v
            return res
        return x
     ```
    And pass it as `object_hook` to `json.loads`.
1. Use the following JSON structure for this configuration:
    ```json
    [
        [null, 2],
        [1.9, 0.5],
        [2,1],
        [3,2],
        [5,3],
        [5.1, 5]
    ]
    ```
    And then simply convert it with `dict()`.

I believe that the first approach is cleaner, though. What do you think?

Reply:
Yes, I think the first approach looks better